src/nrc/datamodel/models.py

```diff
@@ -84,6 +86,51 @@ class Abonnement(models.Model):
         blank=True,
         help_text=_("Client ID extracted from Auth header"),
     )
+    auth_type = models.CharField(
+        _("authorization type"),
+        max_length=30,
+        blank=True,
+        choices=AuthTypes.choices,
+        default=AuthTypes.api_key,
+        help_text=mark_experimental(
+            _(
+                """
+                Required fields per auth type:
+                * `api_key`: `auth`
+                * `zgw`: `client_id`, `secret`
+                * `oauth2_client_credentials`: `client_id`, `secret`, `oauth2_token_url`
+                """
```

Comment:
This currently doesn't show nicely in the setupconfig docs (you can check this locally by running `cd docs && make html "SPHINXOPTS=-E", and then running `chrome _build/html/index.html` and navigating to Installation > Guides > Configuration > Open Notificaties Configuration (CLI):
<img width="596" height="130" alt="afbeelding" src="https://github.com/user-attachments/assets/f987ab21-afb9-42a0-931f-a6e0d01510e6" />

```suggestion
                "Required fields per auth type:\n"
                "* `api_key`: `auth`\n"
                "* `zgw`: `client_id`, `secret`\n"
                "* `oauth2_client_credentials`: `client_id`, `secret`, `oauth2_token_url`"
```